cyclops-ctrl/pkg/cluster/k8sclient/client.go

```diff
@@ -169,6 +171,34 @@ func (k *KubernetesClient) GetPods(namespace, name string) ([]apiv1.Pod, error)
 	return podList.Items, err
 }
 
+func (k *KubernetesClient) GetStreamedPodLogs(ctx *gin.Context, namespace, container, name string, logCount *int64, logChan chan<- string) error {
```

Comment:
Can you pass only the context of `ctx`. I don't want to import gin stuff into k8s client since those are not connected in any way.